handler/pullRequestHandler.go

```diff
@@ -101,6 +86,57 @@ That's something we need before your Pull Request can be merged. Please see our
 	}
 }
 
+// VerifyPullRequestDescription checks that the PR has anything in the body.
+// If there is no body, an 'invalid' label is added and comment posted to the PR with a link to the contributing guide.
+func VerifyPullRequestDescription(req types.PullRequestOuter, contributingURL string, config config.Config) {
+	ctx := context.Background()
+	token := getAccessToken(config, req.Installation.ID)
+	client := factory.MakeClient(ctx, token, config)
+
+	if req.Action == "opened" {
+		if missingDescription(req.PullRequest) {
+			fmt.Println("Applying invalid label")
+			_, res, assignLabelErr := client.Issues.AddLabelsToIssue(ctx, req.Repository.Owner.Login, req.Repository.Name, req.PullRequest.Number, []string{"invalid"})
+			if assignLabelErr != nil {
+				log.Fatalf("%s limit: %d, remaining: %d", assignLabelErr, res.Limit, res.Remaining)
+			}
+
+			body := `Thank you for your contribution. I've just checked and your Pull Request doesn't appear to have any description.
```

Comment:
Seems consistent with the previous messages we used.

@rgee0 do you have any input on this?

Reply:
I don't quite have that English "personality", so I'd appreciate input into proper phrasing :smile: 
I know Derek's personality is one of his strong points!

Reply:
The two comments are very similar.  Is there a way we can extract the main part and just update the 'problem detail' when using it?

`Thank you for your contribution. I've just checked and <problem detail>.
That's something we need before your Pull Request can be merged. Please see our [contributing guide](` + contributingURL + `).`

Reply:
If it's ok, we could do this in a follow up PR. I think there's more to think about than this one instance.

I may have a solution you can see here: https://play.golang.org/p/FByvDuUR4g2
Not sure where the `struct` would be defined though. I was thinking in a new file within the `handlers` package so that it could be used in other handlers.